docs/code_standards.rst

```diff
@@ -59,9 +83,15 @@ Naming
 * _private_variable
 * _PrivateClass
 
+Furthermore, when writing fixtures, make their scope clear at the beginning of the fixture function's name. An unspecified scope is assumed to be function-level
+
+* org
+* module_org
+* session_org
+
 Style
```

Comment:
Should we altogether remove it as black and pre-commit take cares of it automatically about styling.

Reply:
I can remove this